examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p><em>﻿</em>Research domain consists of various learning rate schedulers. They can be compound, meaning you can compose several different schedulers (as a list of dicts).</p><p> </p><p>Scheduler applies weight decay during training on specified iterations. It requires different parameters depending on type.</p><p> </p><p>Some of parameters are meant to control scheduler on BatchFlow level:</p><ul><li><code>'name'</code> — name or alias for a chosen decay&nbsp;(e.g.&nbsp;<strong>'exp'</strong>&nbsp;-&nbsp;<a href="https://pytorch.org/docs/stable/optim.html?highlight=cyclic#torch.optim.lr_scheduler.ExponentialLR" target="_blank">torch.optim.lr_scheduler.ExponentialLR</a>,&nbsp;<strong>'CyclicLR'</strong>&nbsp;-&nbsp;<a href="https://pytorch.org/docs/stable/optim.html?highlight=cyclic#torch.optim.lr_scheduler.CyclicLR" target="_blank">torch.optim.lr_scheduler.CyclicLR</a>);</li><li><code>'frequency'</code> — how often decay is applied;</li><li><code>'first_iter'</code> — when decay starts (default is 0);</li><li><code>'last_iter'</code> — when decay stops (default is -1).</li></ul><p>All other parameters are passed directly to chosen scheduler.</p><p> </p><p>~~~</p><p> </p><p>Also, it's unclear why <code>'exp'</code> used to address <code>'torch...ExponentialLR'</code> , but <code>'CyclicLR'</code>  for <code>'torch...CyclicLR'</code>.  Explain alias mechanics here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>It is not weight decay!</p>